packaging/windows/amazon-cloudwatch-agent-ctl.ps1

```diff
@@ -500,8 +511,24 @@ Function CWATestEC2() {
     return !$error
 }
 
-Function main() {
+Function SetLogLevelAll() {
+    switch -exact ($LogLevel) {
+        INFO { }
+        DEBUG { }
+        ERROR { }
+        WARN { }
+        OFF { }
+        default {
+            Write-Output "Invalid log level: ${LogLevel}`n${UsageString}"
+            Exit 1
+        }
+    }
+
+    & cmd /c "`"${CWAProgramFiles}\amazon-cloudwatch-agent.exe`" --setenv CWAGENT_LOG_LEVEL=${LogLevel} --envconfig ${ENV_CONFIG} 2>&1"
+    CheckCMDResult "" "Set CWAGENT_LOG_LEVEL to ${LogLevel}"
+}
 
+Function main() {

```

Comment:
I'm out of practice with powershell. If we wrap this in a main function, does that maintain the existing run functionality? Pulling from knowledge about Python, just declaring a main function doesn't necessarily run it when you call the script.

Reply:
This was actually already here, but yeah the existing run functionality is preserved.